packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/JSON/ComplexJSONRenderer.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { Typography } from '@mui/material'
+import ExpandCollapseButton from '../../ExpandCollapseButton'
+import { chromeLight, Inspector } from 'react-inspector'
+
+type ComplexJSONRendererProps = {
+  value: any
+}
+
+/**
+ * Handles rendering any complex JSON value in a Synapse table cell. This generally acts as a fallback renderer if the
+ * JSON data doesn't have a simple structure handled by a different renderer.
+ * @param props
+ * @constructor
+ */
+export function ComplexJSONRenderer(props: ComplexJSONRendererProps) {
```

Comment:
Fallback case for JSON objects that aren't simple (flat arrays or objects of depth 1)

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/a7d5b40f-d35e-4c59-a980-f71c92ecb60a